upgrade-notes/backend_20250312_141320.md

```diff
@@ -0,0 +1,3 @@
+#### use InMemoryCache to store the root category during a request ([#3856](https://github.com/shopsys/shopsys/pull/3856))
+
+- see #project-base-diff to update your project
```

Comment:
I feel that upgrade notes are not needed for fixes made by annotation fixer

Reply:
ok, removed